MovieApp/MyCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MyCell.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 30.03.2023..
+//
+
+import Foundation
+import MovieAppData
+import PureLayout
+
+class MyCell: UICollectionViewCell {
```

Comment:
Add methods createViews, styleViews and defineLayoutForViews, same as in LoginViewController. `MyCell` is a view also.